sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteFragment.kt

```diff
@@ -87,6 +87,14 @@ internal class LatteFragment() : Fragment() {
         get() = mutWebView!!
         set(value) { mutWebView = value }
 
+    // This flag is used to work around an issue of fragment re-creation.
+    // When this fragment is used with androidx.navigation,
+    // this fragment may be onDestroyView(), and then onCreateView() again.
+    // If that happens, the webView may never has its load() called.
+    // However, even with this fix, savedInstanceState was never called, so
+    // the webView will show the initial screen.
+    private var waitWebViewToLoadIsCalledInThisLifecycle: Boolean = false
```

Comment:
Does it means sometimes `onDestroyView` will be called but `onSaveInstanceState` won't call?

Reply:
According to https://developer.android.com/guide/fragments/lifecycle#states
`onSaveInstanceState` is always called before calling `onDestroyView`
So on `onCreateView` the webview should be able to restore the state 🤔